apps/src/lib/node/ledger/tendermint_node.rs

```diff
@@ -50,6 +52,12 @@ pub enum Error {
     RollBack(String),
     #[error("Failed to convert to String: {0:?}")]
     TendermintPath(std::ffi::OsString),
+    #[error("Couldn't write {0}")]
+    CantWrite(&'static str),
+    #[error("Couldn't create {0}")]
+    CantCreate(&'static str),
```

Comment:
I think it would be good to add file paths to these `CantWrite` and `CantCreate` as they can help users to figure out what the issue is